packages/bootstrap/extra/Makefile

```diff
@@ -0,0 +1,26 @@
+TOP := $(dir $(lastword $(MAKEFILE_LIST)))
+
+.PHONY: test install clean
+
+# You might like: make PYTEST_OPTS="-s -v" test
+PYTEST_OPTS := -v
+
+test: export MESOS_IP_DISCOVERY_COMMAND = $(TOP)/tests/ip-detect
```

Comment:
I'd rather make the py.test tests run via the top level tox.ini than another new embedded thing